src/applications/edu-benefits/0994/pages/highTechIndustry.js

```diff
@@ -0,0 +1,102 @@
+import fullSchema from 'vets-json-schema/dist/22-0994-schema.json';
+import _ from 'lodash';
+
+const {
+  currentEmployment,
+  currentHighTechnologyEmployment,
+  currentSalary,
+  // highTechnologyEmploymentType, Add this back in once schema is updated
+} = fullSchema.properties;
+
+export const uiSchema = {
+  'ui:description':
+    'To give us an idea of your experience in the high-tech market, please tell us about your work and training.',
+  currentEmployment: {
+    'ui:title': 'Are you working in a high-tech industry now?',
+    'ui:widget': 'yesNo',
+  },
+  currentHighTechnologyEmployment: {
+    'ui:title':
+      'Have you worked in a high-tech industry in the past couple years?',
+    'ui:widget': 'yesNo',
+    'ui:options': {
+      expandUnder: 'currentEmployment',
+      expandUnderCondition: false,
+    },
+    'ui:required': formData => !_.get(formData, 'currentEmployment', false),
+  },
+  'view:salaryEmploymentTypes': {
+    'ui:options': {
+      hideIf: formData => {
+        const showIf =
+          _.get(formData, 'currentEmployment', false) ||
```

Comment:
- can get rid of this block and return the expression directly e.g. `formData => !(_.get ...`
- should be refactored to fit the semantics of `hideIf`